### PR TITLE
Add margin to right of tab group title

### PR DESCRIPTION
**Pull request by peize-li123**
I confirm that no security/privacy review is needed

Ticket for this issue is https://github.com/brave/brave-browser/issues/29259

Description:
Before this PR, we only added a margin to empty title group headers, but non-empty headers also need some space between the header and its tabs.

Before:
image

After:
IMG_2830

### CODE REVIEW

File: chromium_src/chrome/browser/ui/views/tabs/tab_group_style.cc [hunk 1]
```diff
@@ -17,6 +17,7 @@
 #undef TabGroupStyle
 
 const int TabGroupStyle::kStrokeThicknessForVerticalTabs = 4;
+const int TabGroupStyle::kTitleAdjustment = 2;
```
Comment (sangwoo108) [inline chromium_src/chrome/browser/ui/views/tabs/tab_group_style.cc:20]: As we included the original tab_group_style.cc, we might be able to access [the original constant there](https://source.chromium.org/chromium/chromium/src/+/refs/heads/main:chrome/browser/ui/views/tabs/tab_group_style.cc;l=27;drc=a61b95c63b0b75c1cfe872d9c8cdf927c226046e;bpv=1;bpt=1). if so, we can remove this and use [kTitleAdjustmentForEmptyHeader](https://source.chromium.org/chromium/chromium/src/+/refs/heads/main:chrome/browser/ui/views/tabs/tab_group_style.cc;drc=a61b95c63b0b75c1cfe872d9c8cdf927c226046e;bpv=1;bpt=1;l=27?gsn=kTitleAdjustmentForEmptyHeader&gs=KYTHE%3A%2F%2Fkythe%3A%2F%2Fchromium.googlesource.com%2Fcodesearch%2Fchromium%2Fsrc%2F%2Fmain%3Flang%3Dc%252B%252B%3Fpath%3Dchrome%2Fbrowser%2Fui%2Fviews%2Ftabs%2Ftab_group_style.cc%239nGQ1e0yPV2fAnBFrBpX7l_edVVsYCy8z378Se8Uo1k) = 2; for consistency.

File: chromium_src/chrome/browser/ui/views/tabs/tab_group_style.cc [hunk 2]
```diff
@@ -56,3 +57,8 @@ bool TabGroupStyle::ShouldShowVerticalTabs() const {
 
   return tabs::utils::ShouldShowVerticalTabs(tab_group_views_->GetBrowser());
 }
+
+int TabGroupStyle::GetTitleAdjustmentToTabGroupHeaderDesiredWidth(
+    const std::u16string title) const {
+  return kTitleAdjustment;
```
Comment (sangwoo108) [inline chromium_src/chrome/browser/ui/views/tabs/tab_group_style.cc:63]: I think it'd be helpful if we have a comment about why we're overriding this, e.g.

"Sets spacing between title view and group header even when |title| is empty"

Comment (sangwoo108) [reply chromium_src/chrome/browser/ui/views/tabs/tab_group_style.cc:63]: And I'm curious how this will affect the vertical tab, or does nothing to it?

**Review by sangwoo108 — COMMENTED** [review]
Thanks for the PR!

File: chromium_src/chrome/browser/ui/views/tabs/tab_group_style.cc (hunk 1 above)
Comment (peize-li123) [reply chromium_src/chrome/browser/ui/views/tabs/tab_group_style.cc:20]: Hi, Thanks for checking my PR! This was what I tried before but I did not figure our how to submit my code change in the original tab_group_style.cc. Do you have any clue how I can do that?

File: chromium_src/chrome/browser/ui/views/tabs/tab_group_style.cc (hunk 2 above)
Comment (peize-li123) [reply chromium_src/chrome/browser/ui/views/tabs/tab_group_style.cc:63]: It's not affecting the vertical tabs, and if we can modify the original constant then we don't need to override this anymore

File: chromium_src/chrome/browser/ui/views/tabs/tab_group_style.cc (hunk 1 above)
Comment (sangwoo108) [reply chromium_src/chrome/browser/ui/views/tabs/tab_group_style.cc:20]: I seemed to be unclear about it. I think you got it right about modifying chromium_src/.../tab_group_style.cc . But I think you could do something like,

```
// chromium_src/.../tab_group_style.cc
int TabGroupStyle::GetTitleAdjustmentToTabGroupHeaderDesiredWidth(
  return kTitleAdjustmentForEmptyHeader;  // even this constant is defined in the origianl .cc file, we can use it here, I guess.
)
```

Then, you don't need to declare a new constant.

**Comment by sangwoo108** [comment]
Hi, I've run our CI bots with this PR and it seems that the PR isn't formatted properly. Did you happen to run `npm run format`?

**Comment by bsclifton** [comment]
Closing as stale

**Pull request closed without merging**